quixstreams/sources/__init__.py

```diff
@@ -2,3 +2,4 @@
 from .manager import SourceException
 from .multiprocessing import multiprocessing
 from .csv import CSVSource
+from .kafka import KafkaSource, QuixEnvironmentSource
```

Comment:
Let's move the `QuixEnvironmentSource` to a separate `quix` package so it's easier to follow the structure.